src/app/entities/courses/components/course-item/course-item.component.scss

```diff
@@ -1,62 +1,77 @@
 @import "../../../../../mixins";
 @import "../../../../../colors";
 
-.course-item-container {
+.course-item {
   @include card-style;
   width: 100%;
   margin-bottom: 24px;
   padding: 24px;
   box-sizing: border-box;
+  border: solid 4px transparent;
 
-  .course-item-header {
-    position: relative;
+  &--top-rated {
+    background-color: $yellow-light;
+  }
+
+  .course-item__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
     margin-bottom: 8px;
 
-    .course-title {
-      @include font-semibold;
-      color: $gray-dark-1;
-      margin: 0;
+    .course-item__main-info {
+      display: grid;
+      grid-template-columns: auto auto;
+      column-gap: 10px;
+      align-items: center;
+
+      .course-item__title {
+        @include font-semibold;
+        color: $gray-dark-1;
+        margin: 0;
+      }
     }
 
-    .course-additional-info {
+    .course-item__additional-info {
       @include font-regular;
-      position: absolute;
-      top: 0;
-      right: 0;
       display: grid;
       grid-template-columns: auto auto;
       column-gap: 24px;
       color: $gray-light-4;
       font-size: 14px;
 
-      .course-duration,
-      .course-creation-date {
+      .course-item__duration,
+      .course-item__creation-date {
         display: flex;
         align-items: center;
       }
     }
   }
 
-  .course-item-body {
+  .course-item__body {
     display: grid;
     grid-template-columns: 1fr auto;
     column-gap: 40px;
 
-    .course-item-description {
+    .course-item__description {
       @include font-regular;
       font-size: 14px;
       color: $gray-dark-2;
     }
 
-    .course-item-actions {
+    .course-item__actions {
       display: grid;
       grid-template-columns: auto auto;
       column-gap: 12px;
     }
   }
 
-  .course-item-icon {
+  .course-item__icon {
```

Comment:
https://medium.com/code-wild/a-sensible-bem-based-css-structure-for-angular-bdb029ce4356

Reply:
Thanks for useful link. Good to know.